students/lauraannf/Lesson09/run_mailroom_oo.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 29 11:51:56 2018
+
+@author: Laura.Fiorentino
+"""
+
+
+from mailroom_oo import *
+import sys
+import datetime
+import os
+
+
+donor_list = Donor_List()
+donor_list.add_donation('Frank Reynolds', [10, 20, 50])
+donor_list.add_donation('Dee Reynolds', [25, 100])
+donor_list.add_donation('Dennis Reynolds', [10, 50])
+donor_list.add_donation('Mac McDonald', [25, 35, 20])
+donor_list.add_donation('Charlie Kelly', 0.25)
+
+
+def view_donor(donor_list):
+    name = input('Donor Name?>')
+    while True:
+        if donor_list.is_donor(name) is True:
+            donor_list.donors[name].list_donations
+            yn = input('Would you like to add a new donation? y/n>')
+            if yn == 'y':
+                new_donation = input('Donation Amount?>')
+                donor_list.add_donation(name, new_donation)
+                donor_list.donors[name].list_donations
+            else:
+                quit_program()
+        else:
+            print('no such donor')
+            break
+
+
+def add_donor(donor_list):
+    while True:
+        name = input('Type Donor Name or q to quit?>')
+        if name == 'q':
```

Comment:
This is throwing an error for me : TypeError: quit_program() missing 1 required positional argument: 'donor_list'